ceph.spec.in

```diff
@@ -1609,7 +1610,7 @@ exit 0
 %{_mandir}/man8/rados.8*
 %{_mandir}/man8/radosgw-admin.8*
 %{_mandir}/man8/rbd.8*
-%{_mandir}/man8/rbdmap.8*
+%{_mandir}/man8/rbdmap-generator.8*
```

Comment:
might want to move this line to line 1627 -- right before `%{_unitdir}/rbdmap.target`. so it is closer to `rbdmap.target`.